### PR TITLE
Remove declarative_authorization and move to Can^3

### PR DESCRIPTION
**Pull request by alfiehub**

### CODE REVIEW

File: app/controllers/admissions_admin/admissions_controller.rb [hunk 1]
```diff
@@ -1,21 +1,50 @@
...
+
+  before_action :find_by_id, only: %i[edit update]
+
+  has_control_panel_applet :admin_applet,
+                           if: -> { can? :show, Admission }
 
   def show
-    @admission = Admission.find(params[:id])
-    @my_groups = current_user.my_groups
+    # CHECK IF THIS WORKS wITH MULTIPLE BEFORE MERGE
```
Comment (stianste) [inline app/controllers/admissions_admin/admissions_controller.rb]: Don't forget this before merge.

File: app/controllers/images_controller.rb [hunk 2]
```diff
@@ -26,16 +28,11 @@ def create
     end
   end
 
-  def show
-    @image = Image.find(params[:id])
-  end
+  def show; end
```
Comment (stianste) [inline app/controllers/images_controller.rb:31]: Why has this been removed?

File: app/controllers/images_controller.rb [hunk 3]
```diff
@@ -26,16 +28,11 @@ def create
...
 
-  def edit
-    @image = Image.find(params[:id])
-  end
+  def edit; end
```
Comment (stianste) [inline app/controllers/images_controller.rb:33]: And this.

File: docs/role_system.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+# Samfundets role system
+## Overview
+The role system of Samfundet.no is used grant permissions to our members, allowing them access to various features of Samfundet.no depending on the various roles they hold.
+
+The permissions of each role depends on what type of role it is. Access is enforced using `CanCanCan` (Can^3) abilities, all of which are defined in the `app/abilities` folder.
+
+## Role model
+No pun intended. The `Role` model is defined by `app/models/role.rb` and is merged with the `Role` model from the `Samfundet/SamfundetAuth` repository.
```
Comment (stianste) [inline docs/role_system.md:8]: What pun? Haha.

File: spec/abilities/blog_abilities_spec.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require 'rails_helper'
+require 'cancan/matchers'
+
+describe Blog do
+  describe Ability do
+    # before do
```
Comment (stianste) [inline spec/abilities/blog_abilities_spec.rb]: Remove

**Review by stianste — COMMENTED** [review]
LGTM. We should ask ourselves though: what is the worst thing that could happen if we merge this? Then we should write tests for that and make sure that it doesn't.

File: app/controllers/admissions_admin/admissions_controller.rb (hunk 1 above)
Comment (Anderssorby) [reply app/controllers/admissions_admin/admissions_controller.rb]: So every resource is loaded automatically now? Why don't we need to do an explicit query?

**Review by stianste — APPROVED** [review]
LGTM

**Pull request merged**